src/modules/newList.js

```diff
@@ -1,12 +1,25 @@
-/* eslint-disable linebreak-style */
+let indexNumber = 0;
 
 class NewList {
   constructor() {
     this.array = [];
   }
 
-  add(Obj) {
-    this.array.push = new Obj();
+  add(obj) {
+    const getItem = JSON.parse(localStorage.getItem('todo-list'));
+    if (this.array.length > 0) {
+      indexNumber = getItem[getItem.length - 1].index;
+      indexNumber += 1;
+    }
+    obj.index = indexNumber;
+    this.array.push(obj);
+    localStorage.setItem('todo-list', JSON.stringify(this.array));
+  }
```

Comment:
- [ ] When a task is added its index value should be set to its position on the array. For example, if the first task is added its index should be 1, the second one should be 2...the 10th task should have an index of 10, and so on. Right now your index value starts at 0, instead of 1.